.editorconfig

```diff
@@ -13,3 +13,6 @@ trim_trailing_whitespace = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[tests/**/*.json]
+insert_final_newline = false
```

Comment:
Good catch. Could we default to true instead? Maybe we need to append the final newline ourselves in the snapshots, since they're gonna be kept in git and it's considered a best practice.